Lombiq.VueJs/Assets/Scripts/helpers/rollup-pipeline.js

```diff
@@ -1,72 +1,111 @@
-                output: {
-                    format: 'cjs',
-                },
-                onwarn: (warning, next) => {
-                    if (warning.code === 'THIS_IS_UNDEFINED') return;
-                    next(warning);
-                },
-            };
-
-            if (typeof rollupOptions === 'function') rollupOptions = rollupOptions(fileName, entryPath);
-            if (typeof rollupPlugins === 'function') rollupPlugins = rollupPlugins(fileName, entryPath);
-
-            const options = rollupOptions ? { ...defaultRollupOptions, ...rollupOptions } : defaultRollupOptions;
-
-            if (Array.isArray(rollupPlugins)) {
-                if (Array.isArray(options.plugins)) {
-                    rollupPlugins.forEach(plugin => options.plugins.push(plugin));
-                }
-                else {
-                    options.plugins = rollupPlugins;
-                }
+    function configure(fileName, entryPath) {
+        const defaultRollupOptions = {
+            onwarn: (warning, next) => {
+                if (warning.code === 'THIS_IS_UNDEFINED') return;
+                next(warning);
+            },
+        };
+
+        if (typeof rollupOptions === 'function') rollupOptions = rollupOptions(fileName, entryPath);
+        if (typeof rollupPlugins === 'function') rollupPlugins = rollupPlugins(fileName, entryPath);
+
+        const options = rollupOptions ? { ...defaultRollupOptions, ...rollupOptions } : defaultRollupOptions;
+        options.input = entryPath;
+
+        if (Array.isArray(rollupPlugins)) {
+            if (Array.isArray(options.plugins)) {
+                rollupPlugins.forEach((plugin) => options.plugins.push(plugin));
             }
+            else {
+                options.plugins = rollupPlugins;
+            }
+        }
+
+        return options;
+    }
 
-            return gulp.src(entryPath)
-                .pipe(plumber({
-                    errorHandler: function errorHandler(error) {
-                        // Necessary because the regular error.toString() is a bit broken.
-                        const details = Object.entries(error)
-                            .map((pair) => ({
-                                name: pair[0],
-                                value: ('' + pair[1]) === '[object Object]' ? JSON.stringify(pair[1]) :  ('' + pair[1]),
-                            }))
-                            .filter((pair) => pair.name !== 'plugin' && pair.name !== 'message')
-                            .map((pair) => `    ${pair.name}: ${pair.value}`.replace(/\n/g, '\n    '))
-                            .join('\n')
-
-                        log(`Plumber has found an unhandled error:\nError in plugin "${error.plugin}"\nMessage:\n` +
-                            `    ${error.message}\nDetails:\n${details}`);
+    return Promise.all(filesAndEntryPaths
+        .map(async (pair) => {
+            const { fileName, entryPath } = pair;
```

Comment:
```suggestion
        .map(async ({ fileName, entryPath }) => {
```

Welcome to the future!